src/blocksmith/components/article-block-modal/index.js

```diff
@@ -5,7 +5,7 @@ import Footer from '@triniti/cms/blocksmith/components/article-block-modal/Foote
 import Header from '@triniti/cms/blocksmith/components/article-block-modal/Header.js';
 import SelectArticle from '@triniti/cms/blocksmith/components/article-block-modal/SelectArticle.js';
 import { SwitchField, TextField } from '@triniti/cms/components/index.js';
-import ImagePickerField from '@triniti/cms/plugins/dam/components/image-picker-field/index.js';
+import AssetPickerField from '@triniti/cms/plugins/dam/components/asset-picker-field/index.js';
```

Comment:
@rsumilang prefer to have ImagePickerfield.  like how we do node pickers... it's an ArticlePickerField, rather than a configurable NodePickerField even if the ArticlePickerField is merely importing NodePickerField and and customizing it minimally

Reply:
@gdbrown fair enough, update pushed.